Extensions/xkit_preferences.js

```diff
@@ -74,12 +76,40 @@ XKit.extensions.xkit_preferences = new Object({
 			XKit.tools.add_css(mobile_control_panel, 'mobile_xkit_menu');
 		}
 
+		const react_add_button = async (button) => {
+			const account_label = await XKit.interface.translate("Account");
+			const menu_label = await XKit.interface.translate("Menu");
+
+			const insert = () => {
```

Comment:
`insert` is a little misleading, as a name, since it gets called *every* frame a DOM element changes, but only inserts the button if it's been removed from the page. Maybe `check_and_reinsert = () => {`?

Reply:
Sounds good to me!